packages/simulator/lib/vehicles/vehicle.js

```diff
@@ -214,10 +215,30 @@ class Vehicle extends EventEmitter {
       metersMoved / 1000,
       (date - this.lastPositionUpdate) / 1000 / 60 / 60,
     ]
-    // https://www.naturvardsverket.se/data-och-statistik/klimat/vaxthusgaser-utslapp-fran-inrikes-transporter/
-    // https://www.trafa.se/globalassets/rapporter/2010-2015/2015/rapport-2015_12-lastbilars-klimateffektivitet-och-utslapp.pdf
-    const co2 = (this.weight + this.cargoWeight()) * km * this.co2PerKmKg
+
+    /*
+     * CO2
+     *
+     * https://www.naturvardsverket.se/data-och-statistik/klimat/vaxthusgaser-utslapp-fran-inrikes-transporter/
+     * https://www.trafa.se/globalassets/rapporter/2010-2015/2015/rapport-2015_12-lastbilars-klimateffektivitet-och-utslapp.pdf
+     */
+
+    let co2
+    if (this.vehicleType === 'bus' || this.vehicleType === 'taxi') {
+      co2 = km * this.co2PerKmKg
```

Comment:
So for buses and taxi we no longer account for the cargo (passenger) weight. Do we want to do that or is it enough to calculate using a pure average fuel consumption?

Do we know anything about the buses and taxis in Norrland that is of use here?